photon-core/src/main/java/org/photonvision/vision/target/TargetModel.java

```diff
@@ -127,6 +127,21 @@ public enum TargetModel implements Releasable {
                     new Point3(-Units.inchesToMeters(3), Units.inchesToMeters(3), 0),
                     new Point3(-Units.inchesToMeters(3), -Units.inchesToMeters(3), 0),
                     new Point3(Units.inchesToMeters(3), -Units.inchesToMeters(3), 0)),
+            Units.inchesToMeters(3 * 2)),
+    // 2024 FRC tag. 6.5in inner tag, 8.125 overall
+    kAruco6p5in_36h11(
+            List.of(
+                    new Point3(Units.inchesToMeters(6.5 / 2.0), Units.inchesToMeters(6.5 / 2.0), 0),
+                    new Point3(Units.inchesToMeters(6.5 / 2.0), -Units.inchesToMeters(6.5 / 2.0), 0),
+                    new Point3(-Units.inchesToMeters(6.5 / 2.0), -Units.inchesToMeters(6.5 / 2.0), 0),
+                    new Point3(Units.inchesToMeters(6.5 / 2.0), -Units.inchesToMeters(6.5 / 2.0), 0)),
+            Units.inchesToMeters(3 * 2)),
```

Comment:
Should this be `3.25 * 2` or `3 * 6`

Reply:
That's just height right? It's aesthetic and doesn't super matter

Reply:
If the width is bigger it might look slightly off if it isn't. Might just be better for convention.